### DISCUSSION TITLE
Is the Digital Ocean Droplet upgraded to version 4?

### DISCUSSION THREAD

**Discussion by adam-lab** [mastodon/mastodon#21171] [Q&A]
The [Digital Ocean Droplet page](https://marketplace.digitalocean.com/apps/mastodon) says that it will install version 3.5.3. Does anyone know if that is correct, and if there are plans to upgrade the installer to version 4? 

Sorry if this is not the right place to ask, but their website points here for support (via https://discourse.joinmastodon.org/)

**Comment by mjankowski** [comment]
It appears no one has responded to you in three years, which is somewhat confusing.

That said -- the digital ocean mastodon installable app appears to have been discontinued some time ago - so I think the effective answer is that there are no plans to update, and that going that route is not really viable anymore.
